test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -440,7 +447,6 @@ TEST_F(StatisticsFromXMLProfileTests, XMLConfigurationForStatisticsDataWritersQo
 
 #endif // FASTDDS_STATISTICS
 
-    remove("FASTRTPS_PROFILES.xml");
```

Comment:
The file which is generated and saved by the test is kept in the system after the test finishes. Please ensure to remove the XML file from the system.

Reply:
Using the load_XML_profiles_string() will make it unnecessary.